http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -30,12 +30,14 @@ class MonitorAtBoot[F[_]: Timer](implicit F: Async[F],
           clusters.toList.traverse_ {
             case c if c.status.isMonitored && c.status != RuntimeStatus.Unknown =>
               val r = for {
-                _ <- c.cloudService.process(c.id, c.status).compile.drain
+                tid <- traceId.ask
+                message <- runtimeStatusToMessage(c, tid)
+                // If a runtime is in transition status (Creating, Starting etc), then we're enqueue a pubsub message again
+                _ <- message.traverse(m => publisherQueue.enqueue1(m))
```

Comment:
So this will make Front Leo re-send messages for transition statuses? Back Leo should already start monitoring them at boot time, right? I guess it doesn't hurt to have both checks.

Reply:
it doesn't involve front leo...Publisher stream runs on both front and back leo....We can technically enqueue this directly to subscriber's queue as well...but since currently subscriber is defined in a way that only processes google events, so it'll be a bit more refactoring to do that.

Reply:
Ah I see - thanks for the clarification